chalice/app.py

```diff
@@ -121,7 +137,13 @@ def wrapper(*args, **kwargs):
                     chalice_response = future.result(timeout=time_remaining_s)
                     return chalice_response
                 except TimeoutError:
-                    return timeout_response()
+                    if determine_internal_method(method):
+                        # used for "/internal/{endpoint}" methods
+                        request_method, request_path = determine_internal_method(method)
```

Comment:
`determine_internal_method` should not be called twice.

Reply:
Fixed.